modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js

```diff
@@ -90,8 +90,8 @@ const SidebarPanelInfoView = ({
 
 					<p className="component-subtitle">{subType}</p>
 
-					{versions.map((version) => (
-						<div key={version}>
+					{versions.map((version, index) => (
+						<div key={index}>
```

Comment:
`version.version` should be unique, isn't it?

Reply:
Yes, I'll change it 😄